nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/src/main/java/org/apache/nifi/hbase/HBaseClientService.java

```diff
@@ -116,6 +117,13 @@
      */
     void delete(String tableName, byte[] rowId) throws IOException;
 
+    /**
+     * Deletes a list of rows in HBase. All cells are deleted.
+     *
+     */
+
```

Comment:
This blank line can be removed. Also, it might be obvious but I'd expect the same level of documentation with the other methods for Parameters and Exception.

Reply:
Done